gabc/details.html

```diff
@@ -121,7 +121,7 @@ <h3 id="signs">Rhythmic signs</h3>
 <p>In some scores, there are sometimes two levels of horizontal episema: a short episema beneath a longer one. To put an horizontal episemus under a note, type <strong><code>&nbsp;_0&nbsp;</code></strong>. This can be combined with a normal horizontal episemus.</p>
```

Comment:
I'll change out "episemus" for "episema" but will wait for other opinions before worrying about the singular/plural issue.